### ISSUE TITLE
Mouse pointer is overscaled in high dpi

### ISSUE THREAD

**Issue by aaubry** [gitextensions/gitextensions#5424]
**Do you want to request a *feature* or report a *bug*?**

This looks like a bug. I hope this is not a duplicate. I searched and could not find this issue reported anywhere.

**What is the current behavior?**

When using 150% dpi scaling, hovering over some UI elements make the mouse pointer huge, as can be seen in the screenshot below. It seems to happen when the UI element uses a custom mouse cursor.

![Screenshot](https://cld.pt/dl/download/90cac5dc-eed6-4e03-83c7-4524f20532c1/2018-09-13%2009_26_31-Git%20Extensions.png)

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

Move the mouse pointer over an element with a custom pointer, such as: links and resizing handles.

**What is the expected behavior?**

The mouse cursor should be properly scaled. Ideally, it would use the one provided by the operating system, when available.

**Environment you encounter the issue:**

- GitExtensions version: 2.51.05
- GIT version:  2.17.1.windows.2
- OS version: Windows 10 Pro, version 1803, build 17134.228
- .NET version: .NET Framework 4.7.2

**Did this work in previous version of GitExtensions (which)?**

I don't know.

If necessary, I can install other versions to test, or test different builds.

**Comment by vbjay** [comment]
But you can REALLY see it.  I mean you can be across the room and see that
pointer.  Let's call it a hidden feature.  :D

**Comment by aaubry** [comment]
True. The problem is that the cursor flickers when it switches from an arrow (normally sized) to a jumbo hand. Another option would be to always scale up the pointers :P
